mon-pix/tests/integration/components/navbar-header-test.js

```diff
@@ -37,15 +37,16 @@ describe('Integration | Component | navbar-header', function() {
           }
         }
       });
-      await render(hbs`{{navbar-header media=media burger=burger }}`);
+      await render(hbs`<NavbarHeader @burger={{this.burger}}></NavbarHeader>`);
```

Comment:
En one line, j'ai tendance à préférer qui me parait plus lisible
```suggestion
      await render(hbs`<NavbarHeader @burger={{this.burger}} />`);
```